extension/js/common/composer.ts

```diff
@@ -1707,6 +1718,20 @@ export class Composer {
     }
   }
 
+  private async checkEmailAliases() {
+    if (!this.urlParams.isReplyBox) {
+      const addresses = Value.arr.unique((await Settings.fetchAcctAliasesFromGmail(this.urlParams.acctEmail)).concat(this.urlParams.acctEmail));
+      const storedAdresses = (await Store.getAcct(this.urlParams.acctEmail, ['addresses'])).addresses!;
+      if (addresses.sort().join() !== storedAdresses.sort().join()) { // This way of comparation two arrays works only for not object arrays
```

Comment:
If the forced `!` expression above was actually undefined, here you would get an error.